pages/directory/[orgName].tsx

```diff
@@ -60,3 +51,36 @@ const OrganizationPage = () => {
 };
 
 export default OrganizationPage;
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const db = new DbProvider();
+  const clubs = await db.getAllClubs();
+  if (!clubs)
+    return {
+      paths: [],
+      fallback: false,
+    };
+  const paths = clubs.map((club) => ({
+    params: { orgName: club.name },
+  }));
+
+  return { paths, fallback: false };
+};
+
+export const getStaticProps: GetStaticProps<{ club: Club }> = async (ctx) => {
+  const db = new DbProvider();
+  const orgName = ctx?.params?.orgName as string;
+  const clubs = await db.getClubsByName(orgName);
+  const club = clubs[0];
```

Comment:
It looks like the code is assuming that the org the user selected will always be the first item in the results list returned from the API call. This is likely true most of the time, though it looks like there could be a case where this isn't true - does this still work in a scenario where we had clubs named "Org 1" and "Org 2" where the user selects "Org 2"?

Reply:
I was thinking of asking in the next team meeting if we want to move away from using the org names and instead using the ids that firebase gives us. That would eliminate the need for us to search by name whenever loading the page. Just a thought

Reply:
Also, this issue may not arise because when we load the page, we are passing in the full name of the organization that the user chose, unless a club has an identical name, then there shouldn't be any issues

Reply:
In general, I think it would be a good idea to support loading a club by ID. To be sure I've got it, this implementation will return the correct org based on matching the entire name?

Reply:
Yes

Reply:
Sounds good. I'll resolve this for now, though we should make support for fetching clubs by ID a relatively new to-do